### DISCUSSION TITLE
Is there any way to make brakeman fail on obsolete warnings?

### DISCUSSION THREAD

**Discussion by mak-dunkelziffer** [presidentbeef/brakeman#1884] [General]
It would be nice to immediately prune warnings, when they become obsolete. I checked `brakeman -h`, but found no command line flag for this. Is it currently supported to `exit 1` with obsolete warnings? Do you think that this is a reasonable feature or do you think this would make the usage more annoying and think that pruning when you have to edit the ignore-file anyways is sufficient?

**Comment by presidentbeef** [comment]
I don't have a strong opinion here. Given we have `--ensure-ignore-notes` I don't see why we couldn't have `--exit-on-obsolete`.
